modules/mw.cx.TargetArticle.js

```
@@ -230,18 +230,20 @@ mw.cx.TargetArticle.prototype.publishSuccess = function (response, jqXHR) {
 			console.log('LinkToWikidata: ' + JSON.stringify(publishResult.LinkToWikidata));
+			// LinkToWikidata: {"result":"success","qid":"Q474070"}
+			wd_result = publishResult.LinkToWikidata.result;
 		}
 
 		this.translation.setTargetURL(targeturl);
 		var done = this.publishComplete(publishResult.targettitle || null);
 
-		// TODO:
 		if (this.sourceLanguage === "mdwiki") {
-			mw.cx.TargetArticle.prototype.addMdwikiLinks(this.targetLanguage, this.getTargetTitle(), qid)
+			mw.cx.TargetArticle.prototype.addMdwikiLinks(this.targetLanguage, this.getTargetTitle(), qid, wd_result)
```

ESLint

[error] There must be a space after this paren.

(space-in-parens)